### PR TITLE
Document test script

### LINKED ISSUE #2240: Document test script
### What should be done?

https://github.com/FerretDB/FerretDB/tree/main/build/legacy-mongo-shell is not documented at all, but it should be (in CONTRIBUTING.md) because that would allow us to have better bug reports.

We should also update https://github.com/FerretDB/FerretDB/blob/main/.github/ISSUE_TEMPLATE/bug.yml to mention that.

### PR DESCRIPTION
**Pull request by chilagrow**
# Description

Closes #2240.

## Readiness checklist

* [ ] I added/updated unit tests.
* [ ] I added/updated integration/compatibility tests.
* [ ] I added/updated comments and checked rendering.
* [ ] I made spot refactorings.
* [x] I updated user documentation.
* [ ] I ran `task all`, and it passed.
* [x] I ensured that PR title is good enough for the changelog.
* [x] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Assignee, Labels, Project and project's Sprint fields.
* [x] I marked all done items in this checklist.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 1]
```diff
@@ -24,6 +24,14 @@ body:
...
     validations:
       required: true
+  - type: textarea
+    id: how
+    attributes:
+      label: How can we reproduce it?
+      description: >
+        Include a pull request with test script noted in [reporting a bug](../../CONTRIBUTING.md#reporting-a-bug) if you can.
+    validations:
+      required: true
```
Comment (rumyantseva) [inline .github/ISSUE_TEMPLATE/bug.yml:34]: As the previous sentence says, "if you can", maybe this field shouldn't be required?

Comment (chilagrow) [reply .github/ISSUE_TEMPLATE/bug.yml:34]: good point. Let's make it not required!

**Review by rumyantseva — APPROVED** [review]
LGTM

**Review by w84thesun — DISMISSED** [review]
👍

**Review by rumyantseva — APPROVED** [review]

**Review by Fashander — APPROVED** [review]

**Review by AlekSi — APPROVED** [review]
:shipit:

**Pull request merged**